tutorials/getting_started/rules_advanced.md

```diff
@@ -208,6 +338,30 @@ var endTime = time.ZonedDateTime.now().withHour(23).withMinute(0).withSecond(0).
 now.isAfter(sunset) && now.isBefore(endTime)
 ```
 
+:::
+
+::: tab JRuby
+
+```ruby
+Time.now.between? Sunset.state.to_s.."23:00"
```

Comment:
```suggestion
Time.now.between?(Sunset.state.to_s.."23:00")
```

While Ruby (mostly) doesn't care if you use parentheses or not when calling methods, the general  consensus is to not use them for "macro" like calls (`after`, our triggers, etc. more likely fall under that category), and to do use them for all other calls. Predicates should definitely have them.

Reply:
Thank you. Changed